app/models/generic_file.rb

```diff
@@ -4,4 +4,19 @@ class GenericFile < ActiveFedora::Base
   apply_schema IIIFPageSchema
 
   validates_with ViewingHintValidator
+  makes_derivatives :create_access_copy
+
+  private
+
+    def create_access_copy
```

Comment:
It's a small thing, but if we were putting this in the repo, I think we'd be calling this an ["Intermediate File"](https://github.com/duraspace/pcdm/blob/master/pcdm-ext/use.rdf#L28). Maybe rename this method to `:create_intermediate_file`?  ( I realize it's private, but the `makes_derivatives` call near the top caught my eye).